crates/test-macro/src/lib.rs

```diff
@@ -89,7 +89,15 @@ pub fn wasm_bindgen_test(
     // We generate a `#[no_mangle]` with a known prefix so the test harness can
     // later slurp up all of these functions and pass them as arguments to the
     // main test harness. This is the entry point for all tests.
-    let name = format!("__wbgt_{}_{}", ident, CNT.fetch_add(1, Ordering::SeqCst));
+    let internal_id = ident
+        .to_string()
+        .strip_prefix("r#")
+        .map_or(ident.clone(), |s| Ident::new(s, Span::call_site()));
+    let name = format!(
+        "__wbgt_{}_{}",
+        internal_id,
+        CNT.fetch_add(1, Ordering::SeqCst)
+    );
```

Comment:
```suggestion
    let name = format!("__wbgt_{}_{}", ident.unraw(), CNT.fetch_add(1, Ordering::SeqCst));
```

See https://docs.rs/syn/2.0.28/syn/struct.Ident.html#method.unraw.

Reply:
Oh that's wonderful! Will update soon!

Reply:
@daxpedda unfortunately what we get back is a `proc_macro2::Ident` not a `syn::Ident`. There's no `unraw` method for the `proc_macro2` [version](https://docs.rs/proc-macro2/1.0.66/proc_macro2/struct.Ident.html), and `syn` is not a dependency for the `test-macro` crate so there's no way to convert between the two.

Reply:
There's also [`format_ident!`](https://docs.rs/quote/latest/quote/macro.format_ident.html)

Reply:
Hm, could you just pull in `syn`? It's in the dependency tree anyway.

Reply:
> There's also [`format_ident!`](https://docs.rs/quote/latest/quote/macro.format_ident.html)

That's also a good point, we can just use that instead of the new `Ident` we create in the next line.

Reply:
Thank you both!